macros/secondary_calculations/perform_secondary_calculation.sql

```diff
@@ -1,15 +1,15 @@
-{%- macro perform_secondary_calculation(metric_name, grain, dimensions, calc_config) -%}
+{%- macro perform_secondary_calculation(metric_name, grain, dimensions, calc_config, metric_config_dict) -%}
```

Comment:
Perhaps `metric_config` for consistency with `calc_config`? I.e. not specifying the datatype of the argument